core/templates/dev/head/components/forms/schema_editors/schema_based_list_editor_directive.html

```diff
@@ -25,7 +25,7 @@
 
   <!-- The margin-left is added here to left-align the button with the previous list items. -->
   <div style="height: 30px; margin-top: 4px;">
-    <button ng-show="!isOneLineInput || isAddItemButtonPresent"
+    <button ng-show="!isOneLineInput || isAddItemButtonPresent || (localValue.length && localValue[localValue.length - 1].length)"
```

Comment:
Just to check, what is the difference between the second OR-clause and the third OR-clause? If they are checking identical things, should we get rid of the second one (and all the bookkeeping associated with it)?